docs/Using-Pantheon/RPC-PubSub.md

```diff
@@ -149,26 +149,21 @@ are included in the notifications. Otherwise, the transaction hashes are include
 
 ### Logs
 
-Use the `logs` parameter with `eth_subscribe` to be notified of logs included in new blocks. You can 
-specify a [filter object](../Reference/JSON-RPC-API-Objects.md#filter-options-object) to receive notifications 
-only for logs matching your filter.   
+Use the `logs` parameter with `eth_subscribe` to be notified of [logs](Events-and-Logs.md) included in new blocks. You can 
```

Comment:
same as above - logs are not really "included in new blocks"
How about: to be notified of logs created by transactions included in new blocks